tutorials/examples/preprocess_imgs.py

```diff
@@ -100,6 +106,11 @@
 plt.colorbar(orientation='horizontal')
 plt.title('phase flip')
 
+plt.subplot(2, 3, 3)
+plt.imshow(imgs_ds[..., idm], cmap='gray')
```

Comment:
Should be `imgs_ds[idm]`. Does this actually run?

Reply:
You are right. I am still using the old way. It can use `imgs_ds[idm]` after we changed to row major. Done in new commit.